inst/extdata/installPackages.R

```diff
@@ -34,18 +60,59 @@ installPackages <- function(pkgs, lib, repos = getOption("repos")) {
   }
 }
 
-lib <- Sys.getenv("R_LIBS")
-if (nchar(lib) == 0) {
-  stop("Could not get a value for R_LIBS environment variable")
+nodename <- Sys.info()["nodename"]
```

Comment:
gotcha. I understand your logic but I think, again, sorry to beat a horse, we just need to keep this focused on the server needs at this point